internal/ai/conversation.go

```diff
@@ -60,20 +63,35 @@ func ResumeConversation(
 		tools:        tools,
 		Messages:     history.Messages,
 
-		maxTokens: maxTokens,
+		maxOutputTokens: maxOutputTokens,
+		tokenLimit:      100000, // 100k token limit
 	}
 	return c, nil
 }
 
-// sendMessage is the internal implementation with a boolean parameter to specify caching
+// SendMessage sends a message with caching enabled
 func (cc *Conversation) SendMessage(ctx context.Context, messageContent ...anthropic.ContentBlockParamUnion) (*anthropic.Message, error) {
-	// Always set a cache point. Unsupported cache points, e.g. on content that is below the minimum length for caching,
-	// will be ignored
-	cacheControl, err := getLastCacheControl(messageContent)
-	if err != nil {
-		log.Printf("Warning: failed to set cache point: %s", err)
+	return cc.sendMessage(ctx, true, messageContent...)
+}
+
+// sendMessageWithoutCache sends a message without caching (used for summary requests)
+func (cc *Conversation) sendMessageWithoutCache(ctx context.Context, messageContent ...anthropic.ContentBlockParamUnion) (*anthropic.Message, error) {
+	return cc.sendMessage(ctx, false, messageContent...)
+}
```

Comment:
`sendMessageWithoutCache` is a very thin wrapper around `sendMessage(..., false, ....)`. Let's remove `sendMessageWithoutCache` and have summarization call `sendMessage(..., false, ....)` instead